### PR TITLE
doc: enrich build instructions

### PR DESCRIPTION
**Pull request by SilverBut**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### Description

This PR adds some doc about protobuf issues, and require install `grpc-web` plugin.

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Related Tickets & Documents

N/A

### Mobile & Desktop Screenshots/Recordings
N/A

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [x] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by SilverBut** [comment]
I have read the CLA Document and I hereby sign the CLA

File: README.md [hunk 1]
```diff
@@ -5,18 +5,22 @@ Middleware library for Anytype, distributed as part of the Anytype clients.
...
 
-
-
 ### Install local deps 
 
 #### Mac
-As of 16.01.23 last protobuf version (21.12) broke the JS plugin support, so you can use the v3 branch:
-```
+As of 16.01.23 last protobuf version (21.12) [broke the JS plugin support](https://github.com/protocolbuffers/protobuf-javascript/issues/127), so you can use the v3 branch:
+```bash
```
Comment (fat-fellow) [inline README.md:12]: it can be simply shell

**Review by requilence — CHANGES_REQUESTED** [review]
Hi @SilverBut! Thanks for your first PR on anytype-heart ❤️

I've left some comments on the changes.

File: README.md [hunk 2]
```diff
@@ -5,18 +5,22 @@ Middleware library for Anytype, distributed as part of the Anytype clients.
...
 brew install protobuf@3
+brew link --overwrite protobuf@3
```
[the remark this answers is not in the document]
Comment (SilverBut) [reply README.md:14]: Indeed. But maybe users installed homebrew elsewhere (mine is `/usr/local/opt/protobuf@3/bin`). I'll add a memo to tell developers notice their terminal output.

File: README.md [hunk 3]
```diff
@@ -5,18 +5,22 @@ Middleware library for Anytype, distributed as part of the Anytype clients.
...
 ```
 
-To generate Swift protobuf:
+Also, you need to install `grpc-web` plugin:
+```bash
+brew install protoc-gen-grpc-web
```
[the remark this answers is not in the document]
Comment (SilverBut) [reply README.md]: Removed. It's kind of weird...I was encountering failure when run `make setup-protoc` before I opened this pull request, but it works now after restart and clone repository again. Maybe my environment changed accidentally.

[the remark this answers is not in the document]
Comment (SilverBut) [reply README.md]: Will a check of protoc version in Makefile helpful? To avoid unnecessary errors.

**Pull request closed without merging**